docs/usage.md

```diff
@@ -271,6 +271,10 @@ class Product
 
     // ... other fields
 
+    // NOTE: This is not a mapped field of entity metadata, just a simple property.
```

Comment:
What would be the purpose of this?

Reply:
Your question is about the comment? if yes it's the same as in the [doc](https://github.com/dustin10/VichUploaderBundle/blob/master/docs/usage.md?plain=1#L96) without using the Embedded.

If it's about the PR globally it's to show mapping example for Embedded, for example `fileNameProperty: 'image.name', size: 'image.size'`